zebra-state/src/service/check/anchors.rs

```diff
@@ -3,28 +3,29 @@
 
 use std::{collections::HashMap, sync::Arc};
 
-use zebra_chain::{
-    block::{Block, Height},
-    sprout, transaction,
-};
+use zebra_chain::{block::Height, sprout, transaction::Transaction};
 
 use crate::{
     service::{finalized_state::ZebraDb, non_finalized_state::Chain},
-    PreparedBlock, ValidateContextError,
+    ValidateContextError,
 };
 
 /// Checks the final Sapling and Orchard anchors specified by transactions in this
-/// `prepared` block.
+/// `transactions` iter.
 ///
 /// This method checks for anchors computed from the final treestate of each block in
 /// the `parent_chain` or `finalized_state`.
-#[tracing::instrument(skip(finalized_state, parent_chain, prepared))]
```

Comment:
Nitpick: can you please add instrumentation to each public function in this module? It helps a lot with debugging.

It is ok to skip large fields, or fields that can't be traced.

Reply:
Yeah, that wasn't supposed to be removed, fixed in [`65f39ba`](https://github.com/ZcashFoundation/zebra/pull/5716/commits/65f39ba7be9bd586d3a4762502ad1e56a864acdb). Good catch.